modules/ppcp-wc-gateway/src/Notice/GatewayWithoutPayPalAdminNotice.php

```diff
@@ -125,11 +138,20 @@ public function message(): ?Message {
 
 		$name = $gateway->get_method_title();
 
-		$message = sprintf(
-			$text,
-			$name,
-			admin_url( 'admin.php?page=wc-settings&tab=checkout&section=ppcp-gateway' )
-		);
+		if ( $notice_type === self::NOTICE_DISABLED_CARD_BUTTON ) {
+			$message = sprintf(
+				$text,
+				$url1,
+				$url2
+			);
+		} else {
+			$message = sprintf(
+				$text,
+				$name,
+				admin_url( 'admin.php?page=wc-settings&tab=checkout&section=ppcp-gateway' )
+			);
+		}
+
```

Comment:
`else` could be removed here, in general try to avoid using `else` unless it's strictly necessary:
```
		$message = sprintf(
			$text,
			$name,
			admin_url( 'admin.php?page=wc-settings&tab=checkout&section=ppcp-gateway' )
		);
		
		if ( $notice_type === self::NOTICE_DISABLED_CARD_BUTTON ) {
			$message = sprintf(
				$text,
				$url1,
				$url2
			);
		}
```